flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/WindowTableFunctionTest.scala

```diff
@@ -99,8 +100,10 @@ class WindowTableFunctionTest extends TableTestBase {
         |FROM TABLE(
         | TUMBLE(TABLE v1, DESCRIPTOR(cur_time), INTERVAL '15' MINUTE))
         |""".stripMargin
-    thrown.expectMessage("requires the timecol is a time attribute type, but is TIMESTAMP(3)")
-    thrown.expect(classOf[ValidationException])
+    thrown.expectCause(containsCause(
```

Comment:
for most plan test, we should use verifyExecPlan to verify the whole plan result. If we deliberately want to verify the rel plan (with expected traits), use verifyRelPlan